File: samples/Communication/prize/public/cpp/prize.cpp
#include "prize.h"

int find_best(int n) {
	for(int i = 0; i < n; i++) {
		std::vector<int> res = ask(i);
		if(res[0] + res[1] == 0)
			return i;
	}
	return 0;
}

File: samples/Communication/prize/public/pas/prize.pas
unit prize;

interface
uses graderlib;

function find_best(n: longint): longint;

implementation

function find_best(n: longint): longint;
var
	res: array of longint;
	i: longint;
begin
	for i := 0 to n - 1 do
	begin
		res := ask(i);
		if res[0] + res[1] = 0 then
		begin
			find_best := i;
			exit;
		end;
	end;
	find_best := 0;
end;

end.
